### PR TITLE
Handling missing parameter 'ip_version' declaration in "install_iperf3()"

### PR DESCRIPTION
**Pull request by SRIKKANTH**

### CODE REVIEW

File: Testscripts/Linux/utils.sh [hunk 1]
```diff
@@ -2428,6 +2428,13 @@ function install_iperf3 () {
 	ip_version=$1
 	LogMsg "Detected $DISTRO_NAME $DISTRO_VERSION; installing required packages of iperf3"
 	update_repos
+
+#if 'ip_version" is not passed assume its ipv4
+	if [[ "x$ip_version" =~ "x" ]]
+	then
```
Comment (LiliDeng) [inline Testscripts/Linux/utils.sh:2434]: remove ~?

Comment (SRIKKANTH) [reply Testscripts/Linux/utils.sh:2434]: Its tested and working fine.

Comment (LiliDeng) [reply Testscripts/Linux/utils.sh:2434]: image

**Review by atinmu — APPROVED** [review]

File: Testscripts/Linux/utils.sh (hunk 1 above)
Comment (LiliDeng) [reply Testscripts/Linux/utils.sh:2434]: @SRIKKANTH Could you please test in above 3 situations? when ip_version is empty, 4, 6, all will enter into if, that is unexpected.

**Pull request merged**